numpyro/contrib/einstein/kernels.py

```diff
@@ -7,11 +7,12 @@
 import numpy as np
 
 from jax import random
+from jax.lax import stop_gradient
 import jax.numpy as jnp
 import jax.scipy.linalg
 import jax.scipy.stats
 
-from numpyro.contrib.einstein.util import safe_norm, sqrth_and_inv_sqrth
+from numpyro.contrib.einstein.util import median_bandwidth, sqrth_and_inv_sqrth
```

Comment:
Gathered the median bandwidth computation in a single function as it was computed in multiple (some incorrect) ways in the code.